### PR TITLE
Label adopted subcomponents

### PR DESCRIPTION
**Pull request by zkayyali812**
Signed-off-by: Zachary Kayyali <zkayyali@redhat.com>
https://github.com/stolostron/backlog/issues/19185

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -8,11 +8,12 @@ require (
 	github.com/onsi/gomega v1.16.0
 	github.com/openshift/hive/apis v0.0.0-20210910175845-0756b70e6db4
 	github.com/pkg/errors v0.9.1
+	github.com/stolostron/multiclusterhub-operator v0.0.0-20220111202315-a556785ace10
```
Comment (JakobGray) [inline go.mod:11]: I think we should avoid importing github.com/stolostron/multiclusterhub-operator in the backplane. It seems to be adding some ugliness to our dependencies with client-go and mcho is already importing backplane-operator.

**Review by JakobGray — COMMENTED** [review]
We should be able to avoid the need of importing the mch api. The backplane should remain blissfully unaware of mch for the most part. Rather than checking the mch label matches the name we could just assume having the installer labels is sufficient. Or have MCH put on an mce-adopt-me label mce can search for.

**Comment by JakobGray** [comment]
Talked about this some at scrum. Does the `adopted` label get used at all? If nothing references it then it could probably come out.

**Comment by zkayyali812** [comment]
@JakobGray As of now, nothing utilizes the adopted label, it is simply added to the resources. 

Ill update to remove the labels in that case

**Review by JakobGray — APPROVED** [review]

**Pull request merged**